gnovm/tests/files/zrealm_crossrealm17a.gno

```diff
@@ -0,0 +1,132 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	crossrealm "gno.land/r/demo/tests/crossrealm"
+)
+
+type container struct {
+	name string
+	f    *fooer
+}
+
+func (container) Foo() { println("hello " + c.f.name) }
+
+type fooer struct{ name string }
+
+var c = &container{name: "local_container"}
+
+func main() {
+	ff := &fooer{name: "local_foo"}
+	c.f = ff
+	crossrealm.SetFooer(c)
+	crossrealm.CallFooerFoo()
+	print(".")
+}
+
+// Output:
+// hello local_foo
+// .
+
```

Comment:
This test demonstrates how the scenario functions:
<img width="799" alt="image" src="https://github.com/user-attachments/assets/2d41d23c-c008-4157-89a1-4417bb6ed253" />

the `local_container` will be firstly attached by reference to the external realm, after the external realm finalization, the embedding &fooer will be attached to current realm.

Reply:
cc: @moul